ios/Podfile

```diff
@@ -85,6 +85,11 @@ target 'Rainbow' do
       # necessary for Mac Catalyst builds
       :mac_catalyst_enabled => false
     )
+    installer.pods_project.targets.each do |target|
```

Comment:
unsure about this. included it bc somehow `IPHONEOS_DEPLOYMENT_TARGET` was getting set to 11.0 which is incompatible with Xcode 15